### PR TITLE
Added utility functions for inert flesh and gravitational anomaly cores

### PR DESCRIPTION
**Pull request by august-sun**
## About the PR
Adds ability to microwave inert flesh anomaly core for a lot of meat; adds ability for inert gravitational anomaly core to neutralize a singularity.

## Why / Balance
At the moment, anomaly cores aren't really used in anything. This update is one of a planned number of parts to give them functions, with varying degrees of utility for the station.

## Technical details
Edited grav core in relevant YML; added microwave recipe for flesh core, new food, as well as basic recipes, prep details and sprites for said food.

## Media
image
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added new microwave recipe for inert flesh anomaly core.
- add: Added anomalous meat mass. Can be cooked, microwaved, or cut up into a ridiculous amount of beef.
- tweak: Inert gravitational anomaly cores can now completely neutralize a singularity when they come into contact.

### CODE REVIEW

**Comment by mirrorcult** [comment]
grav anom concept is very very good

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml [hunk 1]
```diff
@@ -700,6 +700,35 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: UncookedAnimalProteins
+          Quantity: 90
+        - ReagentId: Fat
+          Quantity: 90
+  - type: SliceableFood
+    count: 10
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:720]: Need more slice doafter time!

File: Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml [hunk 2]
```diff
@@ -1172,6 +1201,35 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 100
+        - ReagentId: Protein
+          Quantity: 50
+  - type: SliceableFood
+    count: 10
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml:1227]: Same

**Review by TheShuEd — APPROVED** [review]

**Comment by TheShuEd** [comment]
Test fails is real

**Comment by august-sun** [comment]
> Test fails is real

Conflicts resolved!

**Comment by tutoumi** [comment]
now blue space anom as a random handled teleporter

**Comment by august-sun** [comment]
> now blue space anom as a random handled teleporter

Looking into it!

**Comment by LittleNorthStar** [comment]
Electrical anomalies could work as high capacity self recharging batteries

**Comment by Spessmann** [comment]
I'm clueless so I'm wondering if by inert you mean the core that's dropped after a anom goes supercritical or when a anom decays?

**Comment by august-sun** [comment]
> I'm clueless so I'm wondering if by inert you mean the core that's dropped after a anom goes supercritical or when a anom decays?

This is the one that drops when the anom decays.

**Comment by NoElkaTheGod** [comment]
Shadow anomaly as a smoke grenade maybe?

**Comment by august-sun** [comment]
Closing for now until I get EVERYTHING I'm looking to have implemented ready!

**Pull request closed without merging**